src/class-elements/private-method-name-property.case

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: PrivateName IdentifierName PropName StringValue
+info: |
+  Static Semantics: PrivateStringValue
+    PrivateName::
+      #IdentifierName
+  Return the String value consisting of the sequence of code units corresponding to PrivateName
+
+template: productions
```

Comment:
As requested in the previous review comment, please use the "default" template. This will produce a test that looks like this: 

```js
var C = class {
  static m() { return 42; } #foo() {};
  method() {
    return this.#foo;
  }
}

var c = new C();

assert.sameValue(C.m(), 42);
assert.sameValue(Object.hasOwnProperty.call(c, "m"), false);
assert.sameValue(Object.hasOwnProperty.call(C.prototype, "m"), false);

verifyProperty(C, "m", {
  enumerable: false,
  configurable: true,
  writable: true,
});

assert.sameValue(c.method().name, '#foo');
```

When it really only needs to produce a test that looks like this: 

```js
var C = class {
  #foo() {};
  method() {
    return this.#foo;
  }
}

var c = new C();

assert.sameValue(c.method().name, '#foo');
```